.github/workflows/status-embed.yml

```
@@ -0,0 +1,56 @@
+name: Status Embed
+
+on:
+  workflow_run:
+    workflows:
+      - Test & Lint
+    types:
+      - completed
+
+jobs:
+  status_embed:
+    if: github.event.workflow_run.conclusion != 'skipped'
+    name:  Send a Status Embed to Discord
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Get Pull Request Information
+        id: pr_info
+        if: github.event.workflow_run.event == 'pull_request'
+        run: |
```

This makes the script safer. `e` and `pipefail` I consider essentially for any non-trivial script to get some sane error handling behaviour.
```suggestion
        run: |
          set -euo pipefail
```

Also consider setting `x`, which will output each command as its executed for debugging purposes.

The `|| exit ...` will no longer be necessary after commands unless you want a custom exit code.